cmake/freertos/common.cmake

```diff
@@ -7,9 +7,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} PARENT_SCOPE)
 function(freertos_fetch_kernel)
     FetchContent_Declare(
         FreeRTOSKernel
-        GIT_REPOSITORY https://github.com/FreeRTOS/FreeRTOS-Kernel
-        GIT_TAG        V10.4.1-kernel-only
-        GIT_PROGRESS   TRUE
+        URL https://github.com/FreeRTOS/FreeRTOS-Kernel/archive/V10.4.1-kernel-only.tar.gz
     )
```

Comment:
I like this. A good permanent solution to avoid this problem occurring once again in the future.